File: c#/MNK2dim/MNK2dim/Measure.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MNK2dim
{
    class Measure
    {
        private double time;
        private double meas;

        public Measure(double time, double meas)
        {
            this.time = time;
            this.meas = meas;
        }

        public double getTime()
        {
            return time;
        }

        public double getMeasure()
        {
            return meas;
        }
     }
}

File: AzimutUgolMesta/measure.cpp
#include "measure.h"

Measure::Measure()
{

}

Measure::Measure(double time, double measure)
{
    this->measure = measure;
    this->time = time;
}

double Measure::getTime()
{
    return this->time;
}

double Measure::getMeasure()
{
    return this->measure;
}
